iLVu2Client/src/Pages/LoginPage.jsx

```diff
@@ -1,12 +1,40 @@
+import React from "react"
+import axios from "axios"
 export default function LoginPage(){
+
+
+    const [userLoginData, setUserLoginData] = React.useState({})
+
+    function handleClick(){
+        axios.post("http://localhost:8000/user/user_login",{
+            ...userLoginData
+        })
+        .then(res=>{
+            console.log(res)
+        }).finally(()=>{
+                // document.location.href = "/journal/tracker"
```

Comment:
Instead of using location.href try to use navigate or other react router hooks it will be better for implementation